src/main/java/com/toofifty/easyblastfurnace/methods/GoldHybridMethod.java

```diff
@@ -43,14 +44,17 @@ public MethodStep next(BlastFurnaceState state)
         // continue doing gold bars until enough coal has been deposited
         // then do one trip of metal bars
 
-        if (state.getInventory().has(ItemID.GOLD_ORE) &&
-            !state.getEquipment().equipped(ItemID.GOLDSMITH_GAUNTLETS)) {
+        if (state.getInventory().has(new int[]{ItemID.SMITHING_CAPE, ItemID.SMITHING_CAPET}) &&
+            !state.getEquipment().equipped(new int[]{ItemID.SMITHING_CAPE, ItemID.SMITHING_CAPET})) {
+            return equipSmithingCape;
+        }
```

Comment:
Maybe this check could be moved to the prerequisite section? I think when I split those steps out I intended for that section to be the "one-time setup" group of steps.

Would there be any use for someone to switch capes 🤔 ? Surely nobody is that sweaty

Reply:
I was wondering about this, I'll put it into prerequisites.